trio/_channel.py

```diff
@@ -1,16 +1,24 @@
 from collections import deque, OrderedDict
 from math import inf
+from typing import NamedTuple
 
 import attr
 from outcome import Error, Value
 
-from .abc import SendChannel, ReceiveChannel, Channel
+from .abc import SendChannel, ReceiveChannel
 from ._util import generic_function, NoPublicConstructor
 
 import trio
 from ._core import enable_ki_protection
 
 
+class MemoryChannelPair(NamedTuple):
+    """Named tuple of send/receive memory channels"""
+
+    send_channel: "MemorySendChannel"
+    receive_channel: "MemoryReceiveChannel"
```

Comment:
I opted for these qualified names rather than just `send` and `receive` to avoid the awkwardness of `send()` and `receive()` also being the primary methods:

```python3
channel_pair = open_memory_channel()
await channel_pair.send.send()
```